### PR TITLE
Update/address rules nic slv

### PR DESCRIPTION
**Pull request by kevinch**
#### What is the purpose of this pull request?
Same as https://github.com/vtex/address-form/pull/247 without beta versions commits.

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/country/SLV.js [hunk 1]
```diff
@@ -0,0 +1,419 @@
...
+    "San Francisco Javier": "3421",
+    "Santa Elena": "3422",
+    "Santa María": "3423",
+    "Santiago de María": "3424",
+    "Tecapán": "3426"
+  }
+}
+
+export default {
+  country: 'SLC',
```
Comment (kevinch) [inline react/country/SLV.js:299]: Shouldn't it be `SLV` here?

Comment (garrucho) [reply react/country/SLV.js]: For sure! 🤦 

Fixed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- Address rules for El Salvador and Nicaragua.
+
+### Fixed
+
+- State/city selection on Guatemala because postalCode was not required when it should.
```
Comment (jeffersontuc) [inline CHANGELOG.md:16]: One change I would make is replace postalCode by postal code or `postalCode` (enforcing this is a property, if thats the case).

File: react/country/SLV.js [hunk 3]
```diff
@@ -0,0 +1,419 @@
+import { TWO_LEVELS } from '../constants'
+import { secondLevelPostalCodes } from '../transforms/postalCodes'
+import { getOneLevel, getTwoLevels } from '../transforms/addressFieldsOptions'
+
+const countryData = {
+  "San Salvador": {
```
Comment (marcoskwkm) [inline react/country/SLV.js:6]: Single quotes here?

Comment (kevinch) [reply react/country/SLV.js:6]: Updated.

File: CHANGELOG.md (hunk 2 above)
Comment (kevinch) [reply CHANGELOG.md:16]: Good point. Updated.

File: react/country/SLV.js [hunk 4]
```diff
@@ -3,296 +3,296 @@ import { secondLevelPostalCodes } from '../transforms/postalCodes'
...
-    "Panchimalco": "1127",
-    "Rosario de Mora": "1128",
-    "San Martín": "1129",
-    "Santiago Texacuangos": "1130",
-    "Santo Tomas": "1131",
-    "Tonacatepeque": "1132"
+  'San Salvador': {
+    'San Salvador': '1101',
+    'San Marcos': '1115',
+    Soyapango: '1116',
```
Comment (marcoskwkm) [inline react/country/SLV.js:9]: I have mixed feelings about some names having quotes and others not having, but that seems to be the standard in this repo so 🤷

Comment (kevinch) [reply react/country/SLV.js:9]: Single-words keys do not need quotes `Soyapango`, whereas multiple-words `San Marcos` keys must. This is a js pattern and handled by Prettier here.

Comment (garrucho) [reply react/country/SLV.js:9]: I always think about why it isn't a common JSON. 🤦

**Pull request merged**